pkg/apis/etcd/v1beta2/cluster.go

```diff
@@ -143,6 +143,14 @@ type PodPolicy struct {
 	// etcd cluster.
 	// The "etcd.version" annotation is reserved for the internal use of the etcd operator.
 	Annotations map[string]string `json:"annotations,omitempty"`
+
+	// busybox init container repository. default is busybox
+	BusyboxRepository string `json:"busyboxRepository,omitempty"`
```

Comment:
Sorry to bother gain.
But after a second though, it would be better and easier to combine BusyboxRepository and BusyboxVersion into one field, e.g. BusyboxImage. Default to "busybox:1.28.0-glibc".